src/components/Theme/style.css

```diff
@@ -33,7 +33,7 @@
 }
 
 .content {
-  flex: 1 0 auto;
+  flex: 0 0 auto;
```

Comment:
The problem now is that the CTA don't sit at the bottom of the content anymore (image one) and the content does not expand (image 2), making the margin between the content and the footer inconsistent.
![screenshot 2019-01-25 at 15 51 56](https://user-images.githubusercontent.com/7127427/51756606-55152b00-20b9-11e9-809c-734370477cd1.png)
![screenshot 2019-01-25 at 15 47 04](https://user-images.githubusercontent.com/7127427/51756617-59d9df00-20b9-11e9-87dd-e0141b8d4b39.png)

Reply:
D'oh - thanks for pointing out. I was just checking if anything looked "broken" rather than comparing things properly to the original SDK 😭